src/components/List/List.js

```diff
@@ -1,40 +1,52 @@
 import React from 'react';
 
-import ReactDragList from 'react-drag-list';
-import 'react-drag-list/assets/index.css';
-
 import './List.css';
 
 class List extends React.Component {
-  _handleUpdate = (evt, updated) => {
-    console.log(evt); // tslint:disable-line
-    console.log(updated); // tslint:disable-line
-    // this.setState({
-    //   dataSource: [...updated, {
-    //     color: '#FFAA00',
-    //     title: 'Added Engineer',
-    //     text: 'Added Engineer',
-    //   }]
-    // })
-  };
+  constructor(props) {
+    super(props);
+
+    this.sortPlayers = this.sortPlayers.bind(this);
+    this.handleInitiativeChange = this.handleInitiativeChange.bind(this);
+  }
+
+  handleInitiativeChange(event, player) {
+    let initiative = event.target.value;
+
+    this.props.handleInitiativeChange(initiative, player);
+    event.preventDefault();
```

Comment:
I don't think it makes a difference, but the convention I've usually seen is to place this at the top of the method.